zerver/openapi/zulip.yaml

```diff
@@ -1088,6 +1088,78 @@ paths:
                                   "name": "test_stream",
                                   "id": 0,
                                 }
+                            - type: object
+                              description: |
+                                Event send when a user_group is added for post permission to
+                                a stream. It is send to all the members of the added user groups
+                                as well as to the user who grants added user groups this permission.
```

Comment:
I think this should be *sent* instead of *send*, with "user_group" inside backticks.